go.mod

```diff
@@ -1,20 +1,19 @@
 module github.com/mailgun/holster/v4
 
-go 1.22.0
+go 1.23.9
```

Comment:
This version is supported starting from https://github.com/golangci/golangci-lint/releases/tag/v1.60.1